new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -199,49 +258,451 @@ public void testDeleteOperatorButNotDomain() {
+    String opReleaseName = OPERATOR_RELEASE_NAME + "1";
+    HelmParams op1HelmParams = new HelmParams().releaseName(opReleaseName)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+    HelmParams opHelmParams = installOperatorHelmChart(opNamespace, opServiceAccount, true, true,
+        true,null,"deployed", 0, op1HelmParams, false, domain1Namespace);
+    int externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
+    String op2ReleaseName = OPERATOR_RELEASE_NAME + "2";
+    HelmParams op2HelmParams = new HelmParams().releaseName(op2ReleaseName)
+        .namespace(op2Namespace)
+        .chartDir(OPERATOR_CHART_DIR);
+
+    // install and verify operator2
+    logger.info("Installing and verifying operator2 fails");
+    try {
+      String expectedError = "Error: Service \"external-weblogic-operator-svc\" "
+          + "is invalid: spec.ports[0].nodePort: Invalid value";
+      HelmParams opHelmParam2 = installOperatorHelmChart(op2Namespace, op2ServiceAccount,
+          true, true, true,
+          expectedError,"failed",
+          externalRestHttpsPort, op2HelmParams, false, domain2Namespace);
+      assertNull(opHelmParam2, "FAILURE: Helm installs operator in the same namespace as first operator installed ");
+    } finally {
+      cleanUpSA(opNamespace);
+      cleanUpSA(op2Namespace);
+      deleteSecret("ocir-secret",opNamespace);
+      deleteSecret("ocir-secret",op2Namespace);
+      uninstallOperator(opHelmParams);
+      uninstallOperator(op2HelmParams);
+    }
+  }
+
+  /**
+   * Install the operator with non existing operator namespace.
+   * The helm install command should fail.
+   *
+   * @throws Exception when helm install does not fail
+   */
+  @Test
+  @DisplayName("Negative test to try to create the operator with not preexisted namespace")
+  @Slow
+  public void testNotPreCreatedOpNsCreateOperatorNegativeInstall() {
+    // install and verify operator
+    logger.info("Installing and verifying operator");
+    String opReleaseName = OPERATOR_RELEASE_NAME + "2";
+    HelmParams op2HelmParams = new HelmParams().releaseName(opReleaseName)
+        .namespace("ns-somens")
+        .chartDir(OPERATOR_CHART_DIR);;
+
+    // install and verify operator
+    logger.info("Installing and verifying operator");
+    String opServiceAccount = op2Namespace + "-sa2";
+    try {
+      String expectedError = "Error: create: failed to create: namespaces \"ns-somens\" not found";
+      HelmParams opHelmParam2 = installOperatorHelmChart("ns-somens", opServiceAccount, false, false,
+          false, expectedError,"failed", 0, op2HelmParams, false, domain2Namespace);
+      assertNull(opHelmParam2, "FAILURE: Helm installs operator in the same namespace as first operator installed ");
+    } finally {
+      uninstallOperator(op2HelmParams);
+    }
+  }
+
+  /**
+   * Install the operator with empty string as domains namespaces
```

Comment:
Missing period at the end of sentence.  

Can we extend the usecase to add a domain namespace by upgrading  the helm chart and make sure the domain is up and running. This will complete the e2e steps described in Quick Start Guide.

Reply:
done

Reply:
Update the javadoc description to reflect the new usecaes i.e addition of new domain namespace and make sure that the the WebLogic domain is activated by Operator.

Reply:
I do not see the change. The description is not complete.

Reply:
corrected